python/cross_service/apigateway_websocket_chat/lambda_chat.py

```diff
@@ -101,7 +102,7 @@ def handle_message(table, connection_id, event_body, apig_management_client):
         logger.exception("Couldn't get connections.")
         status_code = 404
 
-    message = f"{user_name}: {event_body['msg']}".encode("utf-8")
```

Comment:
I don't think we should trust the default encoding, it's a known cross-OS issue if we don't set it explicitly.

Reply:
Based on my [research](https://peps.python.org/pep-0529/), this would only cause an issue if someone is using Windows pre-10:
```
Python 3.6 is already using UTF-8 by default on macOS, Android and Windows
```
Which OS are you concerned about?

Reply:
This is a string literal, which is always created by Python 3 as a UTF-8 string. We've seen the encoding issues when opening a file, which pre 3.10 was opened in an OS + Locale defined way. (In 3.10 the default open() encoding is always `utf-8`.) For this line, it's a NOOP and IMHO ok to clean up.

Reply:
Windows, specifically. And more generally I'm concerned about users who may be using those older problematic versions and copy-pasting this code without knowing it needs an explicit encoding. If you include a comment or there is another way to indicate the encoding required, then I'm fine with that.

Reply:
Added comment.